.pre-commit-config.yaml

```
@@ -146,7 +146,7 @@ repos:
   ## version = re.search('mypy==([0-9\.]*)', open("constraints.txt").read())[1] 
   ## print(f"#========= FROM constraints.txt: v{version} =========")
   ##]]]
-  #========= FROM constraints.txt: v1.5.1 =========
+  #========= FROM constraints.txt: v1.7.1 =========
   ##[[[end]]]
   rev: v1.5.1  # MUST match version ^^^^ in constraints.txt (if the mirror is up-to-date)
```

just to not forget later, if mypy mirror is up-to-date, then we should 
```suggestion
  rev: v1.7.1  # MUST match version ^^^^ in constraints.txt (if the mirror is up-to-date)
```